config/locales/doc_auth/es.yml

```diff
@@ -214,11 +214,9 @@ es:
       secure_account: Vamos a encriptar su cuenta con su contraseña. La encriptación
         significa que sus datos están protegidos y solo usted podrá acceder o
         modificar su información.
-      ssn: Necesitamos su número de Seguro Social para validar su nombre, fecha de
-        nacimiento y habla a. Su número de Seguro Social está encriptado. Con su
-        consentimiento, compartimos este información con la agencia
-        gubernamental a la que está intentando acceder para verificar su
-        identidad.
+      ssn: Necesitamos su número de Seguro Social para verificar su nombre, fecha de
+       nacimiento y habla a.
```

Comment:
this seems like it may be incomplete?

Reply:
Oh interesting, I didn't get this translation because we left the first sentence the same as what's currently live in prod. However I looked at past orders in Gengo, and the Spanish sentence should actually be:

> Necesitamos tu número de Seguro Social para validar tu nombre, fecha de nacimiento y dirección.

Good catch shannon!

Reply:
thank you!